x/staking/simulation/genesis.go

```diff
@@ -52,10 +59,12 @@ func RandomizedGenState(simState *module.SimulationState) {
 
 	simState.AppParams.GetOrGenerate(historicalEntries, &histEntries, simState.Rand, func(r *rand.Rand) { histEntries = getHistEntries(r) })
 
+	simState.AppParams.GetOrGenerate(maximumCommissionRate, &histEntries, simState.Rand, func(r *rand.Rand) { maxCommissionRate = getMaxCommissionRate(r) })
```

Comment:
For some curious reason, `minCommissionRate` is not initialized here. Secondly I don't think you should pass the address og `histEntries` here.